config/m4/fuse3.m4

```diff
@@ -34,15 +45,23 @@ AS_IF([test "x$with_fuse3" != xno],
                       [[#define FUSE_USE_VERSION 30
                         #include <fuse.h>]])
 
-       AC_CHECK_FUNCS([fuse_open_channel fuse_mount fuse_unmount],
-                      [AC_SUBST([FUSE3_LIBS], [$FUSE3_LIBS])],
-                      [fuse3_happy="no"])
+       AS_IF([test "x$fuse3_happy" = "xyes" -a "x$with_fuse3_static" != "xno"],
+             [FUSE3_LIBS="-l:libfuse3.a -lpthread -ldl -lrt"],
```

Comment:
can we obtain "-lpthread -ldl -lrt" `from pkg-config fuse3 --libs --static`?

Reply:
@rakhmets WDYT about thit?

Reply:
Seems `pkg-config --libs-only-l --static fuse3` may fit to the purpose. I'll try it.